pkg/network/ovn_kubernetes.go

```diff
@@ -118,6 +118,11 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data := render.MakeRenderData()
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["OvnImage"] = os.Getenv("OVN_IMAGE")
+	data.Data["OvnNodeImage"] = os.Getenv("OVN_IMAGE")
+	data.Data["OvnMasterImage"] = os.Getenv("OVN_IMAGE")
```

Comment:
Lets call this `OvnControlPlaneImage` to align with the envvar and the naming we've adopted across this feature.

Additionally, the HCP OVN component is called `ovnkube-control-plane` after https://github.com/openshift/cluster-network-operator/pull/1874, not `ovnkube-master`

Reply:
+1
Are you planning to backport this change?